bifacial_radiance/main.py

```diff
@@ -625,13 +751,42 @@ def readWeatherFile(self, weatherFile=None, starttime=None,
         
         
         def _fixStartStop(start_stop,t):
+            '''
```

Comment:
The new starttime and endtime inputs are going to cause big problems.  We should probably plan to be backwards compatible here, meaning that if someone passes '01_01_01' as an input, we should expect that to be January 1 at 1am, not January 1, 2001.    What is the main purpose of this helper function, why was it needed?  Since we're changing the default input behavior it might make sense to accept a range of options here, like datetime.date, datetime.datetime, and pd.Timestamp.  Those would be un-ambiguous.  If someone passes in a string with length 8, we could maybe accept it for now in order to be backwards compatible, but we should throw a warning and say that we're assuming it's in MM_DD_HH format.